README.md

```diff
@@ -27,27 +27,27 @@
     ```
     
     *Note: The SDK uses a standard interface for the http backend see [HttpClient.](https://github.com/evolv-ai/ascend-java-sdk/blob/master/src/main/java/ai/evolv/HttpClient.java). 
-    When configuring the client you must specify what implementation of HttpClient to use (or implement your own). See "Custom Http Backend" for more details.*
+    When configuring the httpClient you must specify what implementation of HttpClient to use (or implement your own). See "Custom Http Backend" for more details.*
```

Comment:
I think your refactor may have been a bit too aggressive here. Some of these httpClients are meant to be ascendClients.
Lines: 30, 39, 43, 123, 125